src/components/InventoryHostStaleness/HostStalenessCard.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState } from 'react';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Flex,
+  Modal,
+  Popover,
+  Tab,
+  TabTitleText,
+  Tabs,
+  Title,
+} from '@patternfly/react-core';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+import TabCard from './TabCard';
+import { CONVENTIONAL_TAB_TOOLTIP, IMMUTABLE_TAB_TOOLTIP } from './constants';
+import { InventoryHostStalenessPopover } from './constants';
+// import { groupsApi } from '../../api';
+
+const HostStalenessCard = () => {
+  //multiply these values be seconds at the end before sending to the api
+  const [filter, setFilter] = useState({
+    system_staleness_delta: '2',
+    system_stale_warning_delta: '14',
+    system_culling_delta: '21',
+    edge_staleness_delta: '7',
+    edge_stale_warning_delta: '150',
+    edge_culling_delta: '120',
+  });
+
+  const [newFormValues, setNewFormValues] = useState(filter);
+  const [edit, setEdit] = useState(false);
+  const [activeTabKey, setActiveTabKey] = useState(0);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isFormValid, setIsFormValid] = useState(true);
+  // const [groupTotal, setGroupTotal] = useState(0);
+
+  const handleTabClick = (_event, tabIndex) => {
+    setActiveTabKey(tabIndex);
+  };
+  const handleModalToggle = () => {
+    setIsModalOpen(!isModalOpen);
+  };
+
+  const resetToOriginalValues = () => {
+    setNewFormValues(filter);
+    setEdit(!edit);
+  };
+
+  const updateHost = () => {};
+
+  // useEffect(() => {
+  //   groupsApi.apiGroupGetGroupList().then((res) => setGroupTotal(res.total));
+  // }, []);
+  // https://console.redhat.com/api/inventory/v1/groups get the count
+  // Need to update the edit button so that it makes the POST req,
+  return (
+    <Card id={'HostStalenessCard'}>
+      <CardHeader>
+        <Title headingLevel="h4" size="xl" id="HostTitle">
+          Organization level system staleness and culling
+        </Title>
+        <InventoryHostStalenessPopover />
```

Comment:
The popover is slightly too far away from the title. Somewhere extra margin-left is added for some reason.

<img width="1504" alt="Screenshot 2023-09-21 at 11 13 32" src="https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/e404ba7e-28f4-4f55-9720-0e46786477a8">

Reply:
It needed to be sm instead of md